packages/polytone/src/ack.rs

```diff
@@ -1,17 +1,27 @@
-use cosmwasm_std::{from_binary, to_binary, Binary, IbcAcknowledgement};
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{from_binary, to_binary, Binary, IbcAcknowledgement, SubMsgResponse};
 
 pub use crate::callback::Callback;
+use crate::callback::{CallbackData, RequestType};
 
 pub type Ack = Callback;
 
+#[cw_serde]
+pub struct InterlError(String);
```

Comment:
There are some errors where we don't know what the type of the callback is, so created this type to match against before executing the callback